static/app/views/performance/traces/content.tsx

```diff
@@ -232,6 +243,10 @@ function TraceRow({
 }) {
   const [expanded, setExpanded] = useState<boolean>(defaultExpanded);
   const [highlightedSliceName, _setHighlightedSliceName] = useState('');
+  const location = useLocation();
+  const queries = useMemo(() => {
+    return decodeList(location.query.query);
+  }, [location.query.query]);
```

Comment:
you could alternatively use `useLocationQuery` which memoizes the query params from the `location` object for you

Reply:
thanks for the tip Ash! I'll prob do that in a follow up